### DISCUSSION TITLE
TNT HD | (Argentina)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#57] [Show and tell]
### Channel Name

TNT HD

### Channel ID

TNTArgentina.ar

### Stream URL

https://edge2-ccast-sl.cvattv.com.ar/live/c3eds/TNT_HD_Arg/SA_SAGEMCOM/TNT_HD_Arg.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by nguoivanchuyen** [comment]
thank you, you can upload full playlist m3u Argentina

**Comment by freearhey** [comment]
<img width="755" alt="image" src="https://user-images.githubusercontent.com/7253922/158994999-3ed9e93f-49ba-4cea-9ea6-7fb155b58ec9.png">

**Comment by BellezaEmporium** [comment]
Works for me, for now.

![snapshot](https://user-images.githubusercontent.com/30985701/159566310-009b9c14-5a74-48ca-9984-cd9d1100a2d6.jpg)

**Comment by R4IN2007** [comment]
https://cdn.prepro.cvattv.com.ar/live/vod/TNTHD/SA_Live_WebVTT_FTA/TNTHD.m3u8

**Comment by freearhey** [comment]
<img width="756" alt="image" src="https://user-images.githubusercontent.com/7253922/185265430-e220a642-544e-4e5f-b123-105d11d26cb8.png">
